pkgs/data/themes/pitch-black/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec{
+  pname = "Pitch-Black";
```

A bit more consistent.
```suggestion
  pname = "pitch-black";
```
That's not a must though, i just think it looks a bit nicer.